tox.ini

```diff
@@ -46,6 +46,7 @@ commands =
   coverage3 run --source circuit_knitting_toolbox --parallel-mode -m pytest test/ {posargs}
   coverage3 combine
   coverage3 html
+  coverage3 report --fail-under=100 --show-missing --include="circuit_knitting_toolbox/circuit_cutting/wire_cutting/wire_cutting_verification.py"
```

Comment:
This looks good. We could even flip it and make 100 the default and point the 85 toward our old tools.